code/modules/client/preference/link_processing.dm

```diff
@@ -853,6 +853,12 @@
 					if(!(brain_type in GLOB.borg_brain_choices))
 						return
 					active_character.cyborg_brain_type = brain_type
+				if("pda_ringtone")
+					var/ringtone = tgui_input_list(user, "What type of ringtone would you like to have on your PDA?", "PDA Ringtones", list("Reset Default Ringtone") + GLOB.pda_ringtone_choices, active_character.pda_ringtone)
+					if(!(ringtone in GLOB.pda_ringtone_choices) || ringtone == "Reset Default Ringtone")
+						active_character.pda_ringtone = null
```

Comment:
This will make it so if you click the cancel button, it will reset your ringtone. Please change this back to how it was the example I gave you.

Reply:
Addressed both this and the db